src/braket_container.py

```diff
@@ -149,14 +150,27 @@ def kick_off_customer_script(entry_point : str) -> multiprocessing.Process:
         str_module, _, str_method = entry_point.partition(":")
         customer_module = importlib.import_module(str_module)
         customer_method = getattr(customer_module, str_method)
-        customer_code_process = multiprocessing.Process(target=customer_method)
+
+        process_kwargs = {"target": customer_method}
+
+        hp_file = os.environ["AMZN_BRAKET_HP_FILE"]
+        with open(hp_file) as f:
+            hyperparams = json.load(f)
```

Comment:
Hyperparameters can also be sent in env variables, do we need to handle both?

Reply:
I believe all user set env variables should be in the AMZN_BRAKET_HP_FILE since this is after the job goes through the service and the customer code is about to be run